### PR TITLE
check unrestricted quota (#5173)

### PR DESCRIPTION
**Pull request by ScharfViktor**
merge commit from master to stable 2.0

### CODE REVIEW

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>wopiValidatorTests-ocis</strong>](https://drone.owncloud.com/owncloud/ocis/17382/41/1) failed. Further test are cancelled...

**Comment by phil-davis** [comment]
> boom Acceptance test [**wopiValidatorTests-ocis**](https://drone.owncloud.com/owncloud/ocis/17382/41/1) failed. Further test are cancelled...

I restarted CI. The wopi server docker image was slow to pull.

**Comment by ScharfViktor** [comment]
This PR is in master branch, I moved it to stable-2.0

File: tests/acceptance/features/bootstrap/SpacesContext.php [hunk 1]
```diff
@@ -1390,7 +1391,7 @@ public function userHasChangedTheQuotaOfTheSpaceTo(
 			"Expected response status code should be 200"
 		);
 	}
-
+
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/SpacesContext.php:1394]: That is a strange space - but it does not matter.

**Review by phil-davis — APPROVED** [review]

**Review by individual-it — APPROVED** [review]

**Pull request merged**